helm/holmes/templates/holmesgpt-service-account.yaml

```diff
@@ -0,0 +1,181 @@
+{{- if .Values.createServiceAccount }}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}-holmes-cluster-role
+  namespace : {{ .Release.Namespace }}
+rules:
+  {{- if .Values.customClusterRoleRules }}
+{{ toYaml .Values.customClusterRoleRules | indent 2 }}
+  {{- end }}
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+      - daemonsets
+      - deployments
+      - events
+      - namespaces
+      - persistentvolumes
+      - persistentvolumeclaims
+      - pods
+      - pods/status
+      - pods/exec
```

Comment:
I would probably add a flag and make this optional (it's a higher permission than the rest and we can do without it for the default toolsets anyway)